packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/Payment/index.tsx

```diff
@@ -112,14 +106,21 @@ const getIcon = (value: SBPaymentMethodType): ReactElement => {
         />
       )
     case 'FUNDS':
-      return <></>
+      return (
+        <Icon
+          size='32px'
+          color='fiat'
+          name={value.currency.toLowerCase() as keyof IcoMoonType}
```

Comment:
👏

Reply:
Yeah it was IcomoonType not DefaultTheme, forgot DefaultTheme is for colors :)